app/controllers/reviews_controller.rb

```diff
@@ -3,12 +3,15 @@ class ReviewsController < ApplicationController
 
   def create
     @video = Video.find(params[:video_id])
-    review = @video.reviews.build(review_params.merge!(user: current_user))
-    if review.save
-      redirect_to @video
+    review = @video.reviews.create(review_params.merge!(user: current_user))
+    if review.errors.any?
+      review.errors.messages.values.each do |msg|
```

Comment:
You could use this `review.errors.full_messages.each do |error|` to return just the array of errors.